synapse/tools/deploy.py

```diff
@@ -25,9 +26,23 @@ def main(argv):
                                                        'does not exist).')
     pars.add_argument('--auth', action='store_true', help='Enable the cell auth subsystem.')
     pars.add_argument('--admin', help='Set the initial <user>:<passwd> as an admin (enables --auth).')
+    pars.add_argument('--module', action='store', help='An additional module to load. This can be used to initialize '
+                                                       'third party module code which may register additional Cells.')
+    pars.add_argument('--cells', action='store_true', help='Print registered cell types and exit.')
 
     opts = pars.parse_args(argv)
 
+    if opts.module:
+        mod = s_dyndeps.tryDynMod(opts.module)
```

Comment:
Can we add a unit test for this before merging?

Reply:
I can do that this afternoon; these updates aren't burning a hole in my pocket atm.

Reply:
added